### DISCUSSION TITLE
Swar plot manager settings help

### DISCUSSION THREAD

**Discussion by brscengiz** [swar/Swar-Chia-Plot-Manager#1030] [Q&A]
Hi guys, the plot does not start with these settings. I’ve been trying for days and couldn’t solve. :pensive:

My computer;
Ryzen 7 3700x (8+8=16)
32gb ram
2x 1TB extreme pro nvme (D:/ and E:/)
and 2xHDD (F:/ and W:/)

My settings;

Global;
max_concurrent: 10
max_for_phase_1: 6
minimum_minutes_between_jobs: 0

name: brscengizchia
max_plots: 999
farmer_public_key:
pool_public_key:
temporary_directory: D:\Plotter
temporary2_directory:
destination_directory: F:\8TB1
size: 32
bitfield: true
threads: 2
buckets: 128
memory_buffer: 4200
max_concurrent: 3
max_concurrent_with_start_early: 4
initial_delay_minutes: 0
stagger_minutes: 60
max_for_phase_1: 3
concurrency_start_early_phase: 4
concurrency_start_early_phase_delay: 0
temporary2_destination_sync: false
exclude_final_directory: false
skip_full_destinations: true
unix_process_priority: 10
windows_process_priority: 32
enable_cpu_affinity: false
cpu_affinity: [ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13 ]

name: brscengizchia2
max_plots: 999
farmer_public_key:
pool_public_key:
temporary_directory: E:\Plotter
temporary2_directory:
destination_directory: W:\24TB
size: 32
bitfield: true
threads: 2
buckets: 128
memory_buffer: 4200
max_concurrent: 3
max_concurrent_with_start_early: 4
initial_delay_minutes: 0
stagger_minutes: 60
max_for_phase_1: 3
concurrency_start_early_phase: 4
concurrency_start_early_phase_delay: 0
temporary2_destination_sync: false
exclude_final_directory: false
skip_full_destinations: true
unix_process_priority: 10
windows_process_priority: 32
enable_cpu_affinity: false
cpu_affinity: [ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13 ]

![Swarhata](https://user-images.githubusercontent.com/85061708/120102641-a388d780-c154-11eb-9bc1-746f4edc85ab.png)

**Comment by brscengiz** [comment]
SOLVED, I made the yaml file again and it was fixed, I think I broke the line. TY.

**Reply by babanana009** [reply]
how many plots on this config dude?

**Reply by brscengiz** [reply]
3+3 
21 plots per day
